{"name": "test_Login[chrome]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <TestCases.TestLogin.TestLogin object at 0x000002D711E7A8B0>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    @pytest.mark.sanity\n    def test_Login(self):\n        self.lp = LoginPage(self.driver)\n>       self.lp.doLogin(self.Email, self.Password)\n\nTestCases\\TestLogin.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nPages\\LoginPage.py:23: in doLogin\n    self.doClick(self.link_Login)\nPages\\BasePage.py:14: in doClick\n    element = WebDriverWait(self.driver, 10).until(EC.visibility_of_element_located(by_locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"16481f6ff0f843cac4b05fd93d297f4f\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x000002D711EB57C0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "parameters": [{"name": "setup", "value": "'chrome'"}], "start": 1619939840128, "stop": 1619939861832, "uuid": "0415aed6-9817-40cb-a2fd-fb24f6c79506", "historyId": "d171ae1be85a63f53ca128b05bd48842", "testCaseId": "bf770cfdb479ed45ee47f0ff8881ac33", "fullName": "TestCases.TestLogin.TestLogin#test_Login", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "TestLogin"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "NLPULTP3403654"}, {"name": "thread", "value": "25924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.TestLogin"}]}